Data transfer updates (#872)
NREL/HPC: docs/Documentation/Managing_Data/Transferring_Files/index.md

rsync is the recommended tool for transferring data between NLR systems. It allows you to easily restart transfers if they fail, and also provides more consistency when dealing with symbolic links, hard links, and sparse files than either scp or cp. It is recommended you do not use compression for transfers within NLR systems. An example command is:

```bash
$ rsync -aP --no-g /scratch/username/dataset1/ /mss/users/username/dataset1/
```

### Transfers to an External Network

!!! Note
    The [Mass Storage System (MSS)](../mss.md) has quotas that limit the number of individual files you can store. If you are copying hundreds of thousands of files then it is best to archive these files prior to copying to Mass Storage. See the [guide on how to archive files](#archiving-files-and-directories).
    
    Mass Storage quotas rely on the group of the file and not the directory path. It is best to use the `--no-g` option when rsyncing to MSS so you use the destination group rather than the group permissions of your source.  You can also `chgrp` your files to the appropriate group prior to rsyncing to MSS.

#### The Data Management Node (Kestrel only)

> [Kestrel](../../Systems/Kestrel/index.md) users are encouraged to transfer data outside of the NLR network through the dedicated data transfer node, `dm1`, whenever possible. `dm1` functions like a login node and can be directly logged into via `ssh <username>@dm1.hpc.nlr.gov`. The `dm1` node is able to access every filesystem on Kestrel and can be used to initiate large data transfers through any of the methods discussed below without impacting the experience of other users on the cluster.

Reviewer: Can you add directions for external users as well?

Author: For sure, but I was not aware it's different for external folks - are they the same directions as we describe for providing a password+OTP?

Reviewer: They won't be able to connect to it from outside of the NLR network, so would either have to first connect to the external login node and then ssh to dm1 from there, or use the HPC VPN

```

### Transfers to an External Network

!!! Note
    The [Mass Storage System (MSS)](../mss.md) has quotas that limit the number of individual files you can store. If you are copying hundreds of thousands of files then it is best to archive these files prior to copying to Mass Storage. See the [guide on how to archive files](#archiving-files-and-directories).
    
    Mass Storage quotas rely on the group of the file and not the directory path. It is best to use the `--no-g` option when rsyncing to MSS so you use the destination group rather than the group permissions of your source.  You can also `chgrp` your files to the appropriate group prior to rsyncing to MSS.

#### The Data Management Node (Kestrel only)

[Kestrel](../../Systems/Kestrel/index.md) users are encouraged to transfer data outside of the NLR network through the dedicated data transfer node, `dm1`, whenever possible. The `dm1` node is able to access every filesystem on Kestrel and can be used to initiate large data transfers through any of the methods discussed below without impacting the experience of other users on the cluster.

!!! Note "Accessing the Data Management Node"

>     The `dm1` node functions like a login node; users who are connected to the internal NLR network may directly login to it from an external location (e.g., from a laptop) via `ssh <username>@dm1.hpc.nlr.gov`, or from another Kestrel node via `ssh dm1`. Any users who are not connected to the internal NLR network will have to first connect to the HPC VPN. Refer to [Accessing Kestrel](../../Systems/Kestrel/index.md#accessing-kestrel) for more information.

Reviewer: ```suggestion
    The `dm1` node functions like a login node; users who are on the NLR network may connect via `ssh <username>@dm1.hpc.nlr.gov`. External users can connect via `ssh dm1` from `kestrel.nlr.gov` or use the SSH gateway or HPC VPN. Refer to [Accessing Kestrel](../../Systems/Kestrel/index.md#accessing-kestrel) for more information.
```
